src/xenia/vfs/devices/disc_image_device.cc

```diff
@@ -30,10 +30,10 @@ DiscImageDevice::~DiscImageDevice() = default;
 bool DiscImageDevice::Initialize() {
   mmap_ = MappedMemory::Open(host_path_, MappedMemory::Mode::kRead);
   if (!mmap_) {
-    XELOGE("Disc image could not be mapped");
+    XELOGE("Disc image {} could not be mapped", xe::path_to_utf8(host_path_));
     return false;
   } else {
-    XELOGFS("DiscImageDevice::Initialize");
+    XELOGFS("DiscImageDevice::Initialize({})", xe::path_to_utf8(host_path_));
```

Comment:
Someone will whine about this change set

Reply:
I can change it back, was just going to revert it to how it was pre-message

Reply:
Personally I don't particularly care. If people do witch hunt then that's their problem. We'll probably catch some rare path bug with it.